Sources/StreamVideoSwiftUI/Livestreaming/LivestreamPlayerViewModel.swift

```diff
@@ -0,0 +1,97 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import StreamVideo
+import SwiftUI
+
+@MainActor
+class LivestreamPlayerViewModel: ObservableObject {
+    @Published private(set) var fullScreen = false
```

Comment:
Where do we have the move to fullscreen logic? I didn't see it

Reply:
we update the value from the view (on tap of the button):
```swift
LivestreamButton(imageName: "viewfinder") {
      viewModel.update(fullScreen: !viewModel.fullScreen)
}
```

Reply:
And what is this doing? I means, is this changing anything on the rendering of the track?

Reply:
hides the controls and notifies the optional closure (provided in the init) for the state change. The player will most likely be part of other views, so customers would need to know when to hide other controls.